src/central-server/admin-service/ui/src/global.ts

```diff
@@ -64,6 +64,8 @@ export enum RouteName {
   SecurityServerAuthenticationCertificates = 'security-server-authentication-certificates',
   SecurityServerAuthenticationCertificate = 'security-server-authentication-certificate',
   SecurityServerClients = 'security-server-clients',
+  AdminUsers = 'admin-users',
```

Comment:
Isn't calling users as admin users bit misleading? There will be observers, registration officers, etc. Users,BackofficeUsers, UiUsers, etc. might represent it better.

This is a general comment, if agreed, it should be applied to all places